addons/hr_holidays/models/calendar_event.py

```diff
@@ -0,0 +1,9 @@
+from odoo import models
+
+
+class CalendarEvent(models.Model):
+    _inherit = 'calendar.event'
+
+    def _need_video_call(self):
+        for record in self:
+            return record.res_model != 'hr.leave'
```

Comment:
It Could be done like this Instead of iterating over each record using a for loop, we can use the filtered() to avoid returning for each record.
```suggestion
    def _need_video_call(self):
        return self.filtered(lambda record: record.res_model != 'hr.leave')
```